src/ColorDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+import { type Color, filterColors } from './utils';
+
+type ColorDisplayProps = {
+    color: Color;
+    isComp?: boolean;
+};
+
+export const ColorDisplay = ({ color: { name, hex, comp }, isComp }: ColorDisplayProps) => {
+    const style = {
+        color: `#${hex}`,
```

Comment:
dark colors are difficult to read against the black background. In general - it's hard to read text that is constantly changing colors. To display a variety of colors, you could consider coloring small div boxes instead.